docs/UserGuide.md

```diff
@@ -39,21 +39,34 @@ StaffConnect (SC) is a **desktop app for managing Professors' and Tutors' contac
 
 --------------------------------------------------------------------------------------------------------------------
 
-
 ## Navigability
 
 ### Overall UI layout
-<br>![Overall Layout](images/OverallLayout.png)
+<br>![UI General](images/UiAnnotationsGeneraL.png)
```

Comment:
Nits, for all the annotations images maybe it is better to input more relevant date and time instead of `11/11/1111 11:11`

Reply:
will amend

Reply:
amended